### DISCUSSION TITLE
Nur eine Info

### DISCUSSION THREAD

**Discussion by hoetzgit** [roe-dl/weewx-DWD#25] [General]
@roe-dl 

Hallo, ich weiß, es passt hier nicht hin. Nur als Info und kann dann gerne gelöscht werden.

Nach der letzten Diskussion um den weewx-mqtt [PR](https://github.com/matthewwall/weewx-mqtt/pull/33) war der Entwickler der Erweiterung [weewx-mqttpublish](https://github.com/bellrichm/weewx-mqttpublish) (auch nach anfänglichem Zögern) von der Sinnhaftigkeit der Anpassungen überzeugt und hat seine entsprechende Extension angepasst. Da Du auf Deiner Webseite auch auf  WeeWX-MQTTSubscribe vom selben Entwickler verweist, dachte ich, Du hättest vielleicht Interesse. Ich habe die Erweiterung bei mir parallel zu weewx-mqtt installiert. Änderungen und Fehlermeldungen sind jederzeit willkommen und werden von Rich zeitnah umgesetzt.

Viele Grüße
Henry

**Comment by roe-dl** [comment]
In der Tat wäre das besser in dem PR untergebracht, auf den es sich bezieht. Wie auch immer, ich würde noch anregen, daß man noch die Möglichkeit hat, das `data_binding` festzulegen. Und ich müßte auf die GPL verweisen, die Quellenverweise verlangt, hier, wo die Idee mit den aggregierten Werten herkommt.

**Reply by hoetzgit** [reply]
Weitere Anregungen gerne an den Entwickler. Ich wollte nur eine Info an Dich geben.
